src/types/bot-interaction.ts

```diff
@@ -11,6 +11,7 @@ export enum BotCommandName {
   SUBTRACT = 'subtract',
   TIMESTAMP = 'timestamp',
   UNIX = 'unix',
+  SNOWFLAKE = 'snowflake'
```

Comment:
This enum is meant to be alphabetically sorted, please insert this new value after `IN`